src/raven.js

```diff
@@ -1174,16 +1174,22 @@ Raven.prototype = {
             }
 
             var fetchInput = args[0];
+
             var method = 'GET';
             var url;
 
             if (typeof fetchInput === 'string') {
               url = fetchInput;
-            } else {
+            } else if (
+              typeof 'Request' !== undefined &&
+              fetchInput instanceof _window.Request
+            ) {
               url = fetchInput.url;
               if (fetchInput.method) {
                 method = fetchInput.method;
               }
+            } else {
+              url = String(fetchInput);
```

Comment:
Can you use `url = '' + url` instead? We use this syntax throughout the codebase.